extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Email\Controller;
+
+use \Email\EmailController;
+use Email\Check;
+
+class CategoryAddController extends EmailController {
+
+	/** @var \Title */
+	private $categoryPage;
+
+	/** @var \Title */
+	private $pageAddedToCategory;
+
+	public function initEmail() {
+		$pageTitle = $this->request->getVal( 'pageTitle' );
+		$titleNamespace = $this->request->getVal( 'namespace', NS_MAIN );
+		$categoryPageId = $this->getVal( 'childArticleID' );
+
+		$this->pageAddedToCategory = \Title::newFromText( $pageTitle, $titleNamespace );
```

Comment:
I'm not sure if it's an issue here, but please check if there isn't possibility to have master/slave replication errors. I suspect this method called in the same request as page is created.